src/Server/Commands/PublicKeyCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace DoSomething\Gateway\Server\Commands;
+
+use JOSE_JWK;
+use Illuminate\Console\Command;
+use DoSomething\Gateway\Common\RestApiClient;
+
+class PublicKeyCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'gateway:key';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Load public key for Gateway server middleware.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $url = config('services.northstar.url');
+        $path = config('services.northstar.key');
+        $http = new RestApiClient($url);
+
+        // Print an error if a URL isn't set in config file.
+        if (empty($url)) {
+            $this->error('Set a Northstar URL in config/services.php! <https://git.io/...>');
+        }
+
+        $discoveryUrl = $url.'/.well-known/openid-configuration';
+
+        $this->comment('Reading configuration from '.$discoveryUrl.'...');
+        $configuration = $http->get($discoveryUrl);
```

Comment:
Do you need to do any error handling if we don't get something back or something goes wrong?

Reply:
Hmm yeah, we should probably add some error handling in case this 404s. 👍

Reply:
Yeah... maybe use a try/catch and incorporate the part below?

Reply:
I added checks for if the route 404s from a URL typo in 8b6b732. I think it's probably okay beyond that since we'd just end up seeing the relevant HTTP error in the console.

Reply:
Ah jes, ok. I might have commented after you already made that change and hadn't realized :|